.browserslistrc

```diff
@@ -1,17 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
Also, why delete this file? Is it no longer used?

Reply:
I don't know, I just ran `ng update` and it deleted it